web/packages/design/src/theme/themes/types.ts

```diff
@@ -87,13 +104,15 @@ export type ThemeColors = {
     textDisabled: string;
     bgDisabled: string;
 
+    /** @deprecated Use `interactive.solid.primary` instead. */
```

Comment:
Oh, that is huge that this now works properly thanks to types in UI components.

Reply:
Unfortunately, only in style functions, but the usage of colors in attributes is somehow limited to just a handful of cases.